server/hotrod/src/main/scala/org/infinispan/server/hotrod/Encoder2x.scala

```diff
@@ -312,6 +312,18 @@ object Encoder2x extends AbstractVersionedEncoder with Constants with Log {
          case s: SizeResponse => writeUnsignedLong(s.size, buf)
          case e: ExecResponse =>
             writeRangedBytes(e.result, buf)
+         case r: IterationStartResponse => writeString(r.iterationId, buf)
+         case r: IterationNextResponse =>
+            writeString(r.iterationId, buf)
```

Comment:
Is writing the iteration id on the response needed?  We should already have the iteration id since we sent the request with it on the client, no?